### ISSUE TITLE
Call scheduler.step() per epoch AND per batch?

### ISSUE THREAD

**Issue by XuMengyaAmy** [aimagelab/meshed-memory-transformer#71]
In your train_xe function, you call scheduler.step() per epoch, then also call scheduler.step() per batch again. Is this your expected training strategy or an accidental mistake?
